collection-scripts/common.sh

```diff
@@ -10,6 +10,7 @@ declare -a DEFAULT_NAMESPACES=(
     "openshift-machine-api"
     "cert-manager"
     "openshift-nmstate"
+    "openshift-marketplace"
```

Comment:
I'm wondering if this should be something that we need to always collect for customers. 
If this is a CI only relevant namespace, I suggest to have this line among the `ADDITIONAL_NAMESPACES` variable [1]

[1] https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/os_must_gather/defaults/main.yml#L27

Reply:
Thank you @fmount for reviewing it. Sounds good to include it in ci-framework.